MM2RandoLib/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MM2Randomizer.Extensions
+{
+    public static class StringExtensions
+    {
+        //
+        // Public Methods
+        //
+
+        public static Byte[] AsIntroString(this String in_String)
+        {
+            if (null != in_String)
```

Comment:
Just a style thing here. Doing a check like this with both branches means the code gets nested a bit. I prefer an early exit on null to avoid that. Like this:

```c#
if (null == in_String) return null;
/* everything that used to be in the body of the if */
```

Reply:
I generally don't like too many early exists, but I don't mind them on parameter checks, so I will change this.